src/psyclone/tests/nemo/nemo_psy_test.py

```diff
@@ -172,6 +171,7 @@ def test_fn_call_no_kernel(parser):
     ''' Check that we don't create a kernel if the loop body contains a
     function call. '''
     reader = FortranStringReader("program fn_call\n"
+                                 "integer, parameter :: wp = kind(1.0)\n"
```

Comment:
Great that this is detected now ... and I expect there will be similar comments for the rest of the files :)